tcms/bugs/static/bugs/js/mutable.js

```diff
@@ -43,10 +51,16 @@ function populateVersion () {
 }
 
 function populateBuild () {
-  const productId = $('#id_version').val()
+  const versionId = $('#id_version').val()
+
+  if (versionId === null) {
+    $('#add_id_build').addClass('disabled')
+  } else {
+    $('#add_id_build').removeClass('disabled')
+  }
 
   const href = $('#add_id_build')[0].href
   $('#add_id_build')[0].href = href.slice(0, href.indexOf('&version'))
-  $('#add_id_build')[0].href += `&version=${productId}`
+  $('#add_id_build')[0].href += `&version=${versionId}`
```

Comment:
It's a bit strange that the URL for version popup ends with `product=null` but the URL for build popup ends with `version=None`. 

Not sure if a problem at the moment.